tests/lib/fake_cloudfront.py

```diff
@@ -507,6 +507,9 @@ def _distribution_config(
             )
         else:
             default_cache_behavior.update({"CachePolicyId": cache_policy_id})
+            default_cache_behavior.pop("DefaultTTL", None)
```

Comment:
I'm puzzled why you're popping out this values here when in https://github.com/cloud-gov/external-domain-broker/pull/423/files#diff-1eee4c4df9a8538fae63c404bedd2d08d98360b41a6c6be6e29ac1c1a5b4c0efR102 you say these are set by the CachePolicy.

Reply:
It is a bit confusing. Basically, I think when you supply a `CachePolicyId`, the managed cache policy sets the values of `MinTTL`, `DefaultTTL`, and `MaxTTL` automatically, so there's a conflict if you're setting those values yourself.

That's why [this documentation](https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_DefaultCacheBehavior.html)says those properties are deprecated when you use cache policies.

And [the documentation on managed cache policies](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-cache-policies.html) outlines the default values that cache policies will set for these properties